### PR TITLE
initial project

### PR DESCRIPTION
**Pull request by justalak**
https://edu-redmine.sun-asterisk.vn/issues/21894

### CODE REVIEW

**Comment by justalak** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>OE28Instagram</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (manhbnt-0960) [inline app/views/layouts/application.html.erb]: i18n, convention

File: test/application_system_test_case.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```
Comment (manhbnt-0960) [inline test/application_system_test_case.rb]: cho test vào git ignore

**Comment by justalak** [comment]
ready

**Pull request merged**